beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2870,6 +2885,20 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         self.remove_notified(&block_root, r)
     }
 
+    pub fn process_gossip_blob_column(
+        self: &Arc<Self>,
+        blob_column: GossipVerifiedBlobColumnSidecar<T>,
+    ) {
+        let blob_column = blob_column.as_blob_column();
+        // TODO(das) send to DA checker
```

Comment:
Until crypto is ready a basic impl could just check that > 50% of columns per row have been seen, without attempting re-construction otherwise